slashCommands/vote.ts

```diff
@@ -31,7 +39,7 @@ export default {
                 minValue: 0
             },
             {
-                type: ApplicationCommandOptionType.Integer,
+                type: ApplicationCommandOptionType.Role,
                 name: "votegroup",
                 description: "The Group that is allowed to vote (0 for @everyone)",
```

Comment:
`0 for @everyone` is still remaining from when this was an integer - since `@everyone` is just selectable now, I think this notice can go